modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -1096,4 +1096,29 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-171627. Verify that that on Target Market, just 1 ~ 3 options can be selected "
+	@priority = 4
+	test ViewTargetMarketOptionsError {
+		property solutions.acceptance = "true";
```

Comment:
Hi @pedroamarall, could you please remove this property? This test shouldn't be executed on solutions acceptance routine

Reply:
Done, thanks!